docs/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+
+
+:zap: Please ensure that you reviewed the [guidelines for contributing](/CONTRIBUTING.md) to this repository. 
+
+
+What does this pull request contain? Please explain your changes.
```

Comment:
We should consider maintaining a changelog (a CHANGELOG.md) and feed directly into it. Then we could just link there and make here a check if the changelog was fed. An example would be the [CHANGELOG.md of the DataSpace Connector](https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/CHANGELOG.md). This needs to go to our [Contributing.md](https://github.com/International-Data-Spaces-Association/idsa/blob/main/CONTRIBUTING.md) like in the [DSC](https://github.com/International-Data-Spaces-Association/DataspaceConnector/blob/main/CONTRIBUTING.md#changelog), they refer to [keepachangelog](http://keepachangelog.com/).

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue